examples/geometry_from_files.py

```diff
@@ -1,8 +1,12 @@
 import flow360 as fl
 from flow360.component.geometry import Geometry
-from flow360.examples import Octahedron
+from flow360.examples import CylinderGeometry
```

Comment:
can we merge this example with this:
https://github.com/flexcompute/Flow360/blob/develop/flow360/examples/cylinder.py ?

Reply:
fixed